frontend/test/__runner__/cypress-runner-utils.js

```diff
@@ -0,0 +1,15 @@
+const chalk = require("chalk");
+
+function printBold(message) {
+  console.log(chalk.bold(message));
+}
+
+function printYellow(message) {
+  console.log(chalk.yellow(message));
+}
+
+function printCyan(message) {
+  console.log(chalk.cyan(message));
+}
```

Comment:
I know this is pretty naive and quite repetitive but tbh I didn't want to invest more time in this at this point, because it doesn't seem we'll use more chalk methods other than these three listed.

If the need for a more versatile solution rises, we can refactor this file.